GLAA.ViewModels/Attributes/RequiredIfAttribute.cs

```diff
@@ -21,7 +21,7 @@ protected override ValidationResult IsValid(object value, ValidationContext cont
                 return ValidationResult.Success;
             }
 
-            return new ValidationResult($"Incorrect usage of {nameof(GetType)}");
+            return new ValidationResult($"Incorrect usage of {GetType()}");
```

Comment:
That would make more sense!

Reply:
That's where that odd `Incorrect usage of GetType` message came from in the validation errors. 

Turns out it errors if you don't have an `ErrorMessage` set